client/state/users/reducer.js

```diff
@@ -26,19 +25,6 @@ export function items( state = {}, action ) {
 			return Object.assign( {}, state, {
 				[ action.user.ID ]: action.user,
```

Comment:
i'm not sure we can store user's globally by ID with Jetpack sites in the mix. What kind of user IDs do we see form Jetpack sites?

Reply:
I agree. Removal of this is part of one of the following PRs I'm going to create which moves the current user from `state/users` to `state/currentUser`. We'll probably end up with a more complex logic to handle different namespaces similar to how we store posts to avoid conflicts.